models/quantal_stp_synapse_impl.h

```diff
@@ -108,8 +76,8 @@ quantal_stp_synapse< targetidentifierT >::set_status( const DictionaryDatum& d,
   updateValue< double >( d, names::u, u_ );
   updateValue< double >( d, names::tau_rec, tau_rec_ );
   updateValue< double >( d, names::tau_fac, tau_fac_ );
-  update_value_int( d, names::n, n_ );
-  update_value_int( d, names::a, a_ );
+  updateValueInt( d, names::n, n_ );
```

Comment:
Why couldn't this just be ``updateValue < int >``? (Same for the next line)

Reply:
Good question! I had not thought about this. I tried now, and got the following error message:
```
error: cannot dynamic_cast ‘(& t)->Token::datum()’ (of type ‘class Datum*’) to type ‘int*’ (target is not pointer or reference to class)
   FT* value = dynamic_cast< FT* >( t.datum() );
```
I will update the docstring of `updateValue`.

Reply:
It looks like ``updateValue< long >()`` works fine here, but using an ``int`` would of course be preferred. @jougs: is there any reason why an overloaded version for ``int``s is not defined for ``getValue()`` in ``sli/tokenutils.cc``? If we can just use ``long``, this whole ``updateValueInt()`` function can go away (which is anyway already casting the ``long`` inside ``IntegerDatum`` to an ``int``).

Reply:
I have updated the comment for `updateValue` now, but agree that not having to use `updateValueInt()` would be good, so I would be happy to change to this.

Reply:
@jougs have you had a chance to look at @clinssen's comment about `updateValue< long >()`?

Reply:
@clinssen @jougs I have now updated to use `updateValue< long >`.